src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/IPackageVulnerabilityService.cs

```diff
@@ -12,5 +12,7 @@ namespace NuGet.PackageManagement.VisualStudio
     public interface IPackageVulnerabilityService
     {
         Task<List<PackageVulnerabilityMetadataContextInfo>> GetVulnerabilityInfoAsync(PackageIdentity packageId, CancellationToken cancellationToken);
+        Task<List<List<PackageVulnerabilityMetadataContextInfo>>> GetVulnerabilityInfoAsync(PackageCollection packageCollection, CancellationToken cancellationToken);
```

Comment:
Why did this signature change to return a list of lists?

Reply:
Because of multiple enumerations.
Nigusu said we had ended up parsing twice in some cases.

Reply:
fwiw, the top method also returns a list, so the list is following an established pattern.